src/Shared/FileUtilities.cs

```diff
@@ -463,6 +463,10 @@ internal static bool LooksLikeUnixFilePath(string value)
             {
                 return true;
             }
+            else if (Directory.Exists(value) || File.Exists(value))
```

Comment:
That's pretty unfortunate, and I think it'd be worth taking a bit of extra care here.

Could the check just be even more explicit for the file-in-root situation?

```c#
if (value[0] == '/' && firstSlash == 0 && (Directory.Exists(value) || File.Exists(value)))
{
    return true;
}
```